.github/workflows/e2e.yml

```
@@ -156,10 +156,50 @@ jobs:
       - name: Run e2e test
         run: timeout 2400 bats "e2e/$E2E_TEST"
 
+  ui_test:
+    runs-on: ${{ matrix.os }}
+    needs: [build_dfx]
+    strategy:
+      matrix:
+        os: [macos-11, ubuntu-20.04]
```

I don't think we need to test across different OSes, right @ericswanson-dfinity ?
```suggestion
        os: [ubuntu-20.04]
```

---

We do need to test this essential functionality across different OSes.